addon/components/webrtc-troubleshooter/template.hbs

```diff
@@ -1,99 +1,119 @@
-                    {{t "webrtcTroubleshoot.checkBandwidth"}} <i class="status-icon pc-moon pc-loading pc-spin"></i>
-                {{else if checkBandwidthSuccess}}
-                    {{t "webrtcTroubleshoot.checkBandwidthSuccess"}} <i class="status-icon pc pc-check"></i>
-                {{else}}
-                    {{t "webrtcTroubleshoot.checkBandwidthFail"}} <i class="status-icon pc pc-close"></i>
-                {{/if}}
-            </div>
-        </div>
-    {{/if}}
-</div>
+    <div class="entry-row">
+      <div class="entry-label">
+        <i class="pc-moon pc-robot-head"></i>
+      </div>
+      <div class="entry-values">
+        {{#if checkingCameraAdvanced}}
+          {{t "webrtcTroubleshoot.checkCameraAdvanced"}} <i class="status-icon pc-moon pc-loading pc-spin"></i>
+        {{else if checkCameraAdvancedSuccess}}
+          {{t "webrtcTroubleshoot.checkCameraAdvancedSuccess"}} <i class="status-icon pc pc-check"></i>
+        {{else}}
+          {{t "webrtcTroubleshoot.checkCameraAdvancedFail"}} <i class="status-icon pc pc-close"></i>
+        {{/if}}
+      </div>
+    </div>
+  {{/if}}
+  <div class="entry-row">
+    <div class="entry-label">
+      <i class="pc pc-signal"></i>
+    </div>
+    <div class="entry-values">
+      {{#if checkingConnectivity}}
+        {{t "webrtcTroubleshoot.checkConnectivity"}} <i class="status-icon pc-moon pc-loading pc-spin"></i>
+      {{else if checkConnectivitySuccess}}
+        {{t "webrtcTroubleshoot.checkConnectivitySuccess"}} <i class="status-icon pc pc-check"></i>
+      {{else}}
+        {{t "webrtcTroubleshoot.checkConnectivityFail"}} <i class="status-icon pc pc-close"></i>
+      {{/if}}
+    </div>
+  </div>
+  <div class="entry-row">
+    <div class="entry-label">
+      <i class="pc pc-throughput"></i>
+    </div>
+    <div class="entry-values">
+      {{#if checkingThroughput}}
+        {{t "webrtcTroubleshoot.checkThroughput"}} <i class="status-icon pc-moon pc-loading pc-spin"></i>
+      {{else if checkThroughputSuccess}}
+        {{t "webrtcTroubleshoot.checkThroughputSuccess"}} <i class="status-icon pc pc-check"></i>
+      {{else}}
+        {{t "webrtcTroubleshoot.checkThroughputFail"}} <i class="status-icon pc pc-close"></i>
+      {{/if}}
+    </div>
+  </div>
+  {{#if video}}
+    <div class="entry-row">
+      <div class="entry-label">
+        <i class="pc pc-bandwidth"></i>
+      </div>
+      <div class="entry-values">
+        {{#if checkingBandwidth}}
+          {{t "webrtcTroubleshoot.checkBandwidth"}} <i class="status-icon pc-moon pc-loading pc-spin"></i>
+        {{else if checkBandwidthSuccess}}
+          {{t "webrtcTroubleshoot.checkBandwidthSuccess"}} <i class="status-icon pc pc-check"></i>
+        {{else}}
+          {{t "webrtcTroubleshoot.checkBandwidthFail"}} <i class="status-icon pc pc-close"></i>
+        {{/if}}
+      </div>
+    </div>
+
+    <div class="bandwidth-stats">
+      {{#if bandwidthStats}}
```

Comment:
Should this `if` block move outside the parent div, so we don't render an empty div when it's disabled?